src/modules/notification/adapters/smsService.adapter.ts

```diff
@@ -84,7 +84,7 @@ export class SmsAdapter implements NotificationServiceInterface {
             const twilio = require('twilio');
             const client = twilio(this.accountSid, this.authToken);
             const message = await client.messages.create({
-                from: `${this.smsFrom}`,
+                from: `+15735704535`,
```

Comment:
@poojakarma Accepting MR for testing purpose as discussed you will send revert Mr after testing